bucket/standardnotes.json

```diff
@@ -25,10 +21,13 @@
         "github": "https://github.com/standardnotes/desktop"
     },
     "autoupdate": {
-        "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win.exe#/dl.7z",
-        "hash": {
-            "url": "$baseurl/latest.yml",
-            "regex": "sha512:\\s+$base64"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win-x64.exe#/dl.7z",
+                "hash": {
+                    "url": "$baseurl/SHA256SUMS"
```

Comment:
Why you use sha256 when there is sha512? Does original au.hash break?

Reply:
The original configuration is pulling the sha512 from a [yaml file](https://github.com/standardnotes/desktop/releases/download/v3.9.7/latest.yml).  The RegEx is capturing the sha512 for the non-x64 exe file which the Standard Notes project doesn't seem to be posting anymore. I didn't see a yaml parser in the autoupdate documents so I figured the default parser for the SHA256SUMS file was less likely to break.  It seemed like I would need a complex RegEx to parse the yaml file.

Reply:
As I was also trying to fix this, I think you can use this regex for the YAML file to keep the same workflow (looking in latest.yml)

````json
    "autoupdate": {
        "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win-x64.exe#/dl.7z",
        "hash": {
            "url": "$baseurl/latest.yml",
            "regex": "\\s+- url: $basename\\s+sha512:\\s+$base64"
        }
    }
````

This works on my side to get the correct hash.